tools/Az.Tools.Predictor/Az.Tools.Predictor/Az.Tools.Predictor.psd1

```diff
@@ -11,7 +11,7 @@
 RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.1.0'
+ModuleVersion = '0.1.2'
```

Comment:
Same concern about version.

Reply:
I revert it back and I'll change the version in a different PR and only in that PR.